src/http-api.js

```diff
@@ -385,24 +391,45 @@ module.exports.MatrixHttpApi.prototype = {
         if (!queryParams) {
             queryParams = {};
         }
-        if (!queryParams.access_token) {
-            queryParams.access_token = this.opts.accessToken;
+        if (this.useAuthorizationHeader) {
+            if (isFinite(opts)) {
+                // opts used to be localTimeoutMs
```

Comment:
It looks like https://github.com/krombel/matrix-js-sdk/commit/9e89e71e0e8d445a31ea6c19b017497e1d26a390#diff-d2cbeec2e0aad0b77c59efdedd123c51L286 implemented the API change.